envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/utils/Ports.kt

```diff
@@ -0,0 +1,22 @@
+package pl.allegro.tech.servicemesh.envoycontrol.utils
+
+import java.net.ServerSocket
+import pl.allegro.tech.servicemesh.envoycontrol.logger
+
+object Ports {
```

Comment:
Maybe it can be moved to test, because it is only used there

Reply:
moving Ports to test causes errors in `envoy-control-tests`, since it has `evony-control-core` as a dependency